package.json

```diff
@@ -13,18 +13,26 @@
   },
   "dependencies": {
     "classnames": "^2.2.3",
-    "object-assign": "^4.1.0"
+    "object-assign": "^4.1.0",
+    "react": "^15.3.0",
```

Comment:
oops, yeah, I moved them here to make eslint shut up but meant to put them in `peerDependencies` :)